src/cli.ts

```diff
@@ -2,13 +2,17 @@
 
 import meow from 'meow'
 import cosmiconfig from 'cosmiconfig'
+import * as fs from 'fs'
 import lynt, { format } from '.'
+import getTSLintConfig from './tslint/config'
+import getESLintConfig from './eslint/config'
 
 const help = `
   Usage
     $ lynt [files] <options>
 
   Options
+    --exportConfig Export tslint or eslint config to use with editors.
```

Comment:
nitpick: could you rename this to `--eject` instead? I think it'll be more inline with other tools that have a similar feature like `create-react-app`.

Reply:
:thinking: I don't know, it is not eject if is revertable, unlike create-react-app wich you can't go back

Reply:
Yes, that is a good point. I think it's fine to leave this as `--exportConfig`.